backend/reservations/management/commands/send_onboarding.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import sys
+from django.core.management.base import BaseCommand, CommandError
+import reservations.config as roombaht_config
+from reservations.models import Guest
+from reservations.helpers import my_url, send_email
+
+logging.basicConfig(stream=sys.stdout, level=roombaht_config.LOGLEVEL)
+logger = logging.getLogger('SendOnboarding')
+
+def onboarding_email(email, otp):
+    hostname = my_url()
+    body_text = f"""
+        BleepBloopBleep, this is the Room Service RoomBaht for Room Swaps letting you know the floors have been cleaned and you have been assigned a room. No bucket or mop needed.
+
+        After you login below you can view your current room, look at other rooms and send trade requests. This functionality is only available until Monday 11/7 at 5pm PST, so please make sure you are good with what you have or trade early.
+
```

Comment:
@otakup0pe @Index01 
Let's put some content in here about how people are expected to navigate the room swap:
```
1. Find your desired room with the same room type (eg queen).
2. Click "SendSwapRequest" and enter in information about how you'd like that room to contact you, eg email address or phone #.
3. They will contact you. Or not.
4. If you both agree to swap rooms, login (link below) and find the room you want to swap under "My Rooms".
5. Click CreateSwapCode. Give this swap code to the other person.
6. Within 10 minutes, they need to go to their page, click "EnterSwapCode" for the room you want, and enter the swap code.
7. Magic intradimensional elves will hop around and inform you of your success. And you will have the room you always wanted!
Really, if you have a friend that you want to swap rooms with, just get their swap code and enter it for your released room.
```

Reply:
Just copied the email text we already had. Not a bad idea to clean it up for this year / clarity. If you get a moment, def push text changes here.

Reply:
pushed updated copy